picard/formats/mp4.py

```diff
@@ -191,11 +191,15 @@ def _load(self, filename):
                     if value.startswith("MusicMagic Fingerprint"):
                         metadata.add("musicip_fingerprint", value[22:])
             elif name == "trkn":
-                metadata["tracknumber"] = values[0][0]
-                metadata["totaltracks"] = values[0][1]
+                if (len(values) >= 1 and len(values[0] >= 1)):
+                    metadata["tracknumber"] = values[0][0]
+                    if (len(values[0])>=2):
+                        metadata["totaltracks"] = values[0][1]
```

Comment:
Two things:
1. Check the placing of parantheses in `len(values[0] >= 1)`
2. Please add spacing around `>=`

I also find this part of the code a bit verbose with the many `len`. What about rewriting it to something like:

```python
elif name == "trkn" and len(values):
    count = len(values[0])
    if count > 0:
        metadata["tracknumber"] = values[0][0]
        if count > 1:
            metadata["totaltracks"] = values[0][1]
```

Same things apply to disk.

Reply:
It would be more pythonic to just catch the raised exception.

Reply:
@zas I tryied that first but i thought that it would be better to use ifs. But I will return to that as I think it will be more easy to read

Reply:
Done.